pennylane/operation.py

```diff
@@ -465,6 +465,12 @@ def generator(self):
         """
         return [None, 1]
 
+    @staticmethod
+    def decomposition(*params, wires=None):
```

Comment:
What do you think of renaming this `decomposition_template`? This comes from me having proposed to have the same template for inversion, too. But if you have this, `RX(x, wires=[0]).inverse()` is rather misleading because it suggests that it does the same as `inv()`. That's why I would name it `inversion_template` in this case. And if one of those has `_template` in the name, all of them should.